docs/Measuring_Resonances.md

```diff
@@ -42,14 +52,24 @@ Recommended connection scheme for I2C:
 | SDA | 03 | GPIO02 (SDA1) |
 | SCL | 05 | GPIO03 (SCL1) |
 
+![MPU-9250 connected to RPI](img/MPU9250-PI-fritzing.png)
```

Comment:
The image links here appear to be broken and do not render in the viewer:
https://github.com/Klipper3d/klipper/blob/04d85a04cd7ea4a0d96fd346c4751f768545e453/docs/Measuring_Resonances.md
*I suspect this has to do with the wrong upper/lower case letters in the name.

Separately, the images seem to be pretty low-res, not sure how they look like in the doc though.

Reply:
Yes they are a-bit blurry. I'll try to generate some new ones.